tests/untl_structure_test.py

```diff
@@ -409,12 +401,12 @@ def test_Metadata_create_xml_string():
     description = us.Description(content=u'Adaption of "Fortuna te dé Dios, hijo"',
                                  qualifier=u'content')
     metadata.children = [title, description]
-    expected_text = """<?xml version="1.0" encoding="UTF-8"?>
+    exp_text = """<?xml version=\'1.0\' encoding=\'utf-8\'?>
 <metadata>
-  <title qualifier="seriestitle">Colecci&#243;n</title>
-  <description qualifier="content">Adaption of "Fortuna te d&#233; Dios, hijo"</description>
+  <title qualifier="seriestitle">Colección</title>
```

Comment:
Was it necessary to change the XML character sequence to the unicode character?

Reply:
i think the way it was before with the ampersand is how it appears on our files on disk, so it would probably be good to  test it.

Reply:
Since strings in python3 are unicode characters, I think that `create_xml_string()` uses unicode on `metadata` root and so used unicode in the `expected_text`.

Reply:
@madhulika95b did you get errors with the way it was before?

Reply:
I've looked into this, and I've drawn some conclusions. The first is that what we _used_ to do I think was technically incorrect, because when we would internally call `tostring()` through `create_xml_string()`, we were not passing an encoding, which meant that the default ASCII encoding was used, and that's why we had some characters being encoded with special XML sequences. Yet we were stating in the declaration that the encoding was UTF-8. With the changes in this PR, we have specifically added the encoding argument to those `tostring()` calls, as well as having that function automatically add the declaration. This means that _now_ the _actual_ encoding matches the _declared_ encoding, and that's also why the returned value (and what _would_ get written to disk), no longer has those XML sequences, but rather the actual unicode characters. Hopefully that clears up what's going on for everyone.

So now that that's out of the way, it seems to me like this is the best approach going forward. Is this how we want it to remain, @ldko, @madhulika95b? I did look over the other comments, and this might have been sort of addressed already, but I'm kind of getting lost on this PR. This asks a similar question to [this](https://github.com/unt-libraries/pyuntl/pull/23#discussion_r286636365) other comment, but just comes at it from another angle.

Reply:
I'm wondering when `tostring()` isn't given any encoding it is using ASCII for encoding by default and returning the value containing ASCII equivalents for the unicode characters and the behavior should be similar when encoding is UTF-8, i-e it should contain UTF-8 characters but this is returning actual unicode characters.

When I tried in python3 interpreter it gave the following:
```
>>> root = etree.XML( '<html><head/><body><p>Hello<br/>Wórld</p></body></html>')
>>> etree.tostring(root)
b'<html><head/><body><p>Hello<br/>W&#243;rld</p></body></html>'
>>> etree.tostring(root, encoding='utf-8')
b'<html><head/><body><p>Hello<br/>W\xc3\xb3rld</p></body></html>'
```

Reply:
So I'm still not certain which way we actually want this to go.

Reply:
I think we want to write utf-8 encoded strings to disk as we are mentioning encoding - UTF-8 in `create_xml_string` function and, the way we have now should be ok.

Reply:
@vphill and @htarver have said this is okay: where pyuntl used to write to disk:
<title qualifier="seriestitle">Colecci&#243;n</title>
we would now write
<title qualifier="seriestitle">Colección</title>?